elizhahn/karaoke-stretch-api
# Refactor/maintenance

## What does this PR do (summary of changes)?
 - The server file was cleaned up and old configurations were removed as well as old route handlers before knex was implemented
 - Two additional route handlers were added to grab a song by id and to grab a song's genres
 - A migration file was added to update the table schema for the genres table. It was missing a unique id column 
 - Additional data was added to the seeding file to flush out the database
## How should it be tested?
- Fetch down the branch and run `knex migration:latest` to run the new migrations
- Run `knex seed:run`  to seed the database with the new data that was added
 -   make an api call to each of the specified endpoints on postman: /songs, /songs/8, /8/genres 
    - You should see song data coming through
## Future Iterations:
 - Build out all CRUD functionality including post, delete, put/patch
